insights-notification-schemas-java/src/main/resources/schemas/Action.json

```diff
@@ -1,7 +1,7 @@
 {
   "type": "object",
   "title": "Notification message",
-  "comment": "Generated with https://hellosean1025.github.io/json-schema-visual-editor/",
+  "$comment": "Generated with https://hellosean1025.github.io/json-schema-visual-editor/",
```

Comment:
What does this `$` change?

Reply:
`$comment` is the accepted keyword for a comment.
https://json-schema.org/understanding-json-schema/reference/generic.html#comments